aws-py-hub-and-spoke-network/requirements.txt

```diff
@@ -1,3 +1,3 @@
 pulumi>=3.0.0,<4.0.0
-pulumi_aws>=5.0.0,<6.0.0
-pulumi_awsx>=1.0.0,<2.0.0
+pulumi_aws>=6.0.0,<7.0.0
+pulumi_awsx==2.0.0b3
```

Comment:
Is there a reason we're specifying what appears to be a beta release here? And pinning the version?

Reply:
An oversight, thanks!